### PR TITLE
remvoe is-in-browser dep

### PR DESCRIPTION
**Pull request by behnammodi**
## What Would You Like to Add/Fix?
remove `is-in-browser` module

## Todo

- All tests passed
- This is not a public API changed

## Expectations on Changes

knowing the environment is so easy and we don't need a module for that

## Changelog

remove `is-in-browser` module

### CODE REVIEW

File: packages/react-jss/src/createUseStyles.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import isInBrowser from 'is-in-browser'
+import {isInBrowser} from 'jss'
```
Comment (kof) [inline packages/react-jss/src/createUseStyles.js:2]: tbh I don't really want this to become a public API of jss

Comment (kof) [reply packages/react-jss/src/createUseStyles.js:2]: I am fine with having it in both packages twice, unless its needed in more places, then it should be a separate package

Comment (behnammodi) [reply packages/react-jss/src/createUseStyles.js:2]: I agree with you I will add it in both of the packages `jss` and `react-jss`

Comment (behnammodi) [reply packages/react-jss/src/createUseStyles.js:2]: If we put it into a separate package, what name should be for this package? something like `jss-utils`?

Comment (kof) [reply packages/react-jss/src/createUseStyles.js]: if we would do that, why not just use that package that we have now?

Comment (behnammodi) [reply packages/react-jss/src/createUseStyles.js]: Actually, I wanted to save a few bytes, but I think this hasn't more worth

Comment (behnammodi) [reply packages/react-jss/src/createUseStyles.js]: I will close this PR, are you agree?

Comment (kof) [reply packages/react-jss/src/createUseStyles.js]: yep

Comment (kof) [reply packages/react-jss/src/createUseStyles.js]: if you want to work on something, let me know and we will figure out a good issue
I recently added a discord if you want to chat https://discord.gg/e79XhqmaDH

Comment (behnammodi) [reply packages/react-jss/src/createUseStyles.js]: Great, my pleasure

**Pull request closed without merging**